mockingbird/build.gradle.kts

```diff
@@ -36,9 +36,16 @@ kotlin {
                 implementation(libs.touchlab.stately.concurrency)
                 implementation(libs.touchlab.stately.concurrent.collections)
                 implementation(libs.kotlinx.coroutines)
+            }
+        }
+
+        val commonTest by getting {
+            dependencies {
                 implementation(libs.kotlin.test)
```

Comment:
kotlin test was a production library :(

Reply:
Thanks

Reply:
keep also in mind that mockingbird is a testing library as well, clients should use mockingbird as a test library